### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by cfortier2**

### CODE REVIEW

File: v1/chronos.service [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+Description=Chronos
+After=docker.service mesos-master.service
+Requires=docker.service mesos-master.service
+
+[Service]
+Environment=CHRONOS_IMAGE=mesosphere/chronos:chronos-2.3.3-1.0.1.ubuntu1404-mesos-0.22.0-1.0.ubuntu1404
+# NOTE: chronos doesn't need the `zk://`
+Environment=ZOOKEEPER=<ZK_ELB>
+
+Restart=on-failure
```
Comment (nBerg) [inline v1/chronos.service]: is this what we want?

File: v1/chronos.service [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ExecStartPre=-/usr/bin/docker kill chronos
+ExecStartPre=-/usr/bin/docker rm chronos
+ExecStartPre=/usr/bin/docker pull ${CHRONOS_IMAGE}
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+    --name chronos \
+    --net=host \
+    ${CHRONOS_IMAGE} \
+    /usr/bin/chronos \
+    run_jar \
+    --http_port 4400 \
```
Comment (nBerg) [inline v1/chronos.service]: do any ports need to be exposed?

File: v1/chronos.service [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+Restart=on-failure
+RestartSec=20
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill chronos
+ExecStartPre=-/usr/bin/docker rm chronos
+ExecStartPre=/usr/bin/docker pull ${CHRONOS_IMAGE}
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+    --name chronos \
+    --net=host \
```
Comment (ko-be) [inline v1/chronos.service]: Do we want net=host?  @cfortier2

File: v1/chronos.service (hunk 1 above)
Comment (cfortier2) [inline v1/chronos.service]: according to http://www.freedesktop.org/software/systemd/man/systemd.service.html , yes.

`Setting this to on-failure is the recommended choice for long-running services, in order to increase reliability by attempting automatic recovery from errors.`

File: v1/chronos.service (hunk 2 above)
Comment (cfortier2) [inline v1/chronos.service]: it's using the host network, so you don't need to expose any ports

File: v1/fleet_boostrap.sh [hunk 4]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# this script exists to bootstrap our Mesos cluster.
```
Comment (ko-be) [inline v1/fleet_boostrap.sh]: Typo in filename "boostrap"

File: v1/chronos.service (hunk 3 above)
Comment (cfortier2) [inline v1/chronos.service]: yes. these are based off of: https://github.com/mesosphere/coreos-setup/blob/master/etc/systemd/system/mesos-master.service#L16

File: v1/fleet_boostrap.sh (hunk 4 above)
Comment (cfortier2) [inline v1/fleet_boostrap.sh]: d'oh

File: v1/mesos-master.service [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+ExecStartPre=/usr/bin/docker pull ${MESOS_IMAGE}
+ExecStart=/usr/bin/sh -c "/usr/bin/docker run \
+    --name=mesos_master \
+    --privileged \
+    --net=host \
+    ${MESOS_IMAGE} \
+    --ip=$(/usr/bin/ip -o -4 addr list eth0 | grep global | awk \'{print $4}\' | cut -d/ -f1) \
+    --zk=zk://${ZOOKEEPER}/mesos \
+    --work_dir=/var/lib/mesos/master \
+    --quorum=1"
```
Comment (nBerg) [inline v1/mesos-master.service]: why one?

**Pull request merged**